libs/npm/core/src/components/grid-table/PagedTable.tsx

```diff
@@ -21,6 +21,7 @@ export const PagedTable = <CT extends object = Record<string, unknown>>({
   return (
     <GridTable
       data={page.items}
+      manualPageSize
```

Comment:
Don't do this.  It will always be true.  Remove this property as it will already be passed down if the implementation requires it through the `{...rest}`

Reply:
Will get to this on Monday.